docs/book/stacks-and-components/stack-deployment/cloud-stacks/minimal-gcp-stack.md

```diff
@@ -0,0 +1,179 @@
+{% endtab %}
+{% endtabs %}
+
+#### Orchestrator
+
+{% tabs %}
+{% tab title="CLI" %}
+```bash
+export ORCHESTRATOR_NAME=gcp_vertex_orchestrator
+
+# Register the GCS artifact-store and reference the target GCS bucket
+zenml orchestrator register ${ORCHESTRATOR_NAME} --flavor=vertex 
+  --project=<PROJECT_NAME> --location=europe-west2
+
+# Connect the GCS orchestrator to the target gcp project via a GCP Service Connector
+zenml orchestrator connect ${ORCHESTRATOR_NAME} -i
+```
+
+{% hint style="info" %}
+Head on over to our [docs](../../component-guide/orchestrators/vertex.md) to learn more about the registration of an orchestrator.
+{% endhint %}
+{% endtab %}
+
+{% tab title="Dashboard" %}
+<figure><img src="../../../.gitbook/assets/Create_Orchestrator.png" alt=""><figcaption><p>Select the Vertex Orchestrator</p></figcaption></figure>
+
+<figure><img src="../../../.gitbook/assets/Register_Orchestrator_Connector.png" alt=""><figcaption><p>Name it, Select the Connector and set an appropriate location. All other fields are optional.</p></figcaption></figure>
+{% endtab %}
+{% endtabs %}
+
+#### Container Registry
+
+{% tabs %}
+{% tab title="CLI" %}
+```bash
+export CONTAINER_REGISTRY_NAME=gcp_container_registry
+
+zenml container-registry register ${CONTAINER_REGISTRY_NAME} --flavor=gcp --uri=<GCR-URI>
+
+# Connect the GCS orchestrator to the target gcp project via a GCP Service Connector
+zenml container-registry connect ${CONTAINER_REGISTRY_NAME} -i
+```
+
+{% hint style="info" %}
+Head on over to our [docs](../../component-guide/container-registries/gcp.md) to learn more about the registration of a container registry.
+{% endhint %}
+{% endtab %}
+
+{% tab title="Dashboard" %}
+<figure><img src="../../../.gitbook/assets/Create_Container_Registry.png" alt=""><figcaption><p>Choose the GCP Container Registry.</p></figcaption></figure>
+
+<figure><img src="../../../.gitbook/assets/Create_Container_Registry_Connector.png" alt=""><figcaption><p>Name it and select the connector.</p></figcaption></figure>
+{% endtab %}
+{% endtabs %}
+
+### 7) Create Stack
+
+
+
+{% tabs %}
+{% tab title="CLI" %}
+```bash
+export STACK_NAME=gcp_stack
+
+zenml stack register ${STACK_NAME} -o ${ORCHESTRATOR_NAME} \
+    -a ${ARTIFACT_STORE_NAME} -c ${CONTAINER_REGISTRY_NAME} --set
+```
+
+{% hint style="info" %}
+In case you want to also add any other stack components to this stack, feel free to do so.
+{% endhint %}
+{% endtab %}
+
+{% tab title="Dashboard" %}
+<figure><img src="../../../.gitbook/assets/Create_Stack.png" alt=""><figcaption><p>Combine the three stack components and you have your minimal GCP stack. Feel free to add any other component of your choice as well.</p></figcaption></figure>
+{% endtab %}
+{% endtabs %}
+
+## And you're already done!
+
+Just like that, you now have a fully working GCP stack ready to go. Feel free to take it for a spin by running a pipeline on it.
```

Comment:
Add a final section: "Clean Up" which explains how to delete the ZenML resources, and how to delete the GCP resources (i.e. maybe a `gcloud` CLI command to delete the whole project + associated resources?)

Reply:
great idea